src/commands/code-blocks/removeCodeBlocks.ts

```diff
@@ -0,0 +1,447 @@
+import { CommandModule } from "yargs";
+import { promises as fs } from "fs";
+import MagicString from "magic-string";
+import { visit } from "../../tree";
+import restructured from "../../restructured";
+import { CanonicalLanguageValues } from "./CanonicalLanguageValues";
+import { LanguageFileExtensions } from "./LanguageFileExtensions";
+import { LanguageValueMappings } from "./LanguageValueMappings";
+import * as path from "path";
+import { LanguageMapper } from "./LanguageMapper";
+import { CodeBlockWithMetadata } from "./CodeBlockWithMetadata";
+import { PageWriteData } from "./PageWriteData";
+import { CodeNode } from "./CodeNode";
+
+export const removeCodeBlocks = async (
```

Comment:
Oops... I think this refactor breaks the auto-discovery of commands in the commands/ directory.
Instead leave this file in `src/commands/`
And move the rest to `src/code-blocks/`